src/firestore/firestore-internal.ts

```diff
@@ -25,14 +25,22 @@ import { App } from '../app';
 
 export class FirestoreService {
 
-  private appInternal: App;
-  private firestoreClient: Firestore;
+  private readonly appInternal: App;
+  private databases: Map<string, Firestore> = new Map();
```

Comment:
I think this also can be `readonly databases: Map<string, Firestore> = new Map()`